
#### Spectral counts differential expression by edgeR

Given a null and an alternative model, with a **two level** treatment factor as the two conditions to compare, executes the negative binomial test by edgeR functions to discover differentially expressed features between the two conditions. The null and alternative models may include blocking factors. The reference level of the main factor is considered to be the control condition.    

#### References

1. Robinson MD, McCarthy DJ and Smyth GK (2010). edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139-140   

2. Robinson MD and Smyth GK (2007). Moderated statistical tests for assessing differences in tag abundance. Bioinformatics 23, 2881-2887   

3. Robinson MD and Smyth GK (2008). Small-sample estimation of negative binomial dispersion, with applications to SAGE data. Biostatistics, 9, 321-332    

4. Josep Gregori, Laura Villareal, Alex Sanchez, Jose Baselga, Josep Villanueva (2013). An Effect Size Filter Improves the Reproducibility in Spectral Counting-based Comparative Proteomics. Journal of Proteomics, DOI http://dx.doi.org/10.1016/j.jprot.2013.05.030    